integrations/apis/flux.weave.works/v1beta1/types.go

```diff
@@ -105,6 +105,10 @@ type HelmReleaseStatus struct {
 	// managed by this resource.
 	ReleaseStatus string `json:"releaseStatus"`
 
+	// Revision would define what Git hash has currently been deployed.
```

Comment:
This could also contain the chart version, if the chart is from Helm, couldn't it?

Reply:
Sure! Does `Revision` make sense for chart version? As in, could one field be used for both? Something tells me no...